pkg/codegen/python/gen_test.go

```diff
@@ -150,6 +150,11 @@ func buildVirtualEnv() error {
 }
 
 func pyTestCheck(t *testing.T, codeDir string) {
+	extraDir := filepath.Join(filepath.Dir(codeDir), "python-extras")
```

Comment:
Thought about this one, but probably reasonable. It really does nothing but run the tests. So if no tests, don't run.